### ISSUE TITLE
OpenTTD 12.2 does not install correctly

### ISSUE THREAD

**Issue by kroppt** [Calinou/scoop-games#645]
Here is the install log:

```
openttd: 1.10.0 -> 12.2
Updating one outdated app:
Updating 'openttd' (1.10.0 -> 12.2)
Downloading new version
openttd-12.2-windows-win64.zip (8.0 MB) [=====================================================================] 100%
Checking hash of openttd-12.2-windows-win64.zip ... ok.
opengfx-7.1-all.zip (3.4 MB) [================================================================================] 100%
Checking hash of opengfx-7.1-all.zip ... ok.
opensfx-1.0.3-all.zip (11.0 MB) [=============================================================================] 100%
Checking hash of opensfx-1.0.3-all.zip ... ok.
openmsx-0.4.2-all.zip (136.9 KB) [============================================================================] 100%
Checking hash of openmsx-0.4.2-all.zip ... ok.
Uninstalling 'openttd' (1.10.0)
Removing shim 'openttd.shim'.
Removing shim 'openttd.exe'.
Removing shim 'openttd.ps1'.
Unlinking ~\scoop\apps\openttd\current
Installing 'openttd' (12.2) [64bit]
Loading openttd-12.2-windows-win64.zip from cache
Loading opengfx-7.1-all.zip from cache
Loading opensfx-1.0.3-all.zip from cache
Loading openmsx-0.4.2-all.zip from cache
Extracting openttd-12.2-windows-win64.zip ... done.
Extracting opengfx-7.1-all.zip ... done.
Extracting opensfx-1.0.3-all.zip ... done.
Extracting openmsx-0.4.2-all.zip ... done.
Running pre_install script...
Move-Item : Cannot find path 'C:\Users\Tyler\scoop\apps\openttd\12.2\opengfx-0.6.0.tar' because it does not exist.
At line:1 char:1
+ Move-Item "$dir\opengfx-0.6.0.tar" "$dir\baseset"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\Tyler\...engfx-0.6.0.tar:String) [Move-Item], ItemNotFoundExce
   ption
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.MoveItemCommand

Move-Item : Cannot find path 'C:\Users\Tyler\scoop\apps\openttd\12.2\opensfx-0.2.3' because it does not exist.
At line:2 char:1
+ Move-Item "$dir\opensfx-0.2.3" "$dir\baseset"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\Tyler\...2\opensfx-0.2.3:String) [Move-Item], ItemNotFoundExce
   ption
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.MoveItemCommand

Move-Item : Cannot find path 'C:\Users\Tyler\scoop\apps\openttd\12.2\openmsx-0.3.1' because it does not exist.
At line:3 char:1
+ Move-Item "$dir\openmsx-0.3.1" "$dir\baseset"
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\Tyler\...2\openmsx-0.3.1:String) [Move-Item], ItemNotFoundExce
   ption
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.MoveItemCommand

Linking ~\scoop\apps\openttd\current => ~\scoop\apps\openttd\12.2
Creating shim for 'openttd'.
Creating shortcut for OpenTTD (openttd.exe)
'openttd' (12.2) was installed successfully!
```

Firstly, it was still a successful install, even though it failed.

Secondly, the pre_install script wasn't updated for the new asset versions.

**Comment by Calinou** [comment]
> Firstly, it was still a successful install, even though it failed.

This is how Scoop works. You need to uninstall the app to try installing it again.

**Comment by kroppt** [comment]
> I've pushed a fix in [fabfd49](https://github.com/Calinou/scoop-games/commit/fabfd49f503bd7e881e9a74e5ecf020d75943d27), can you try it?

Yes, the installation works now.
